makefile

```diff
@@ -0,0 +1,8 @@
+
+.PHONY: markdownlint markdownlint-fix
+
+markdownlint:
+	docker run -v $(CURDIR):/workdir --rm  ghcr.io/igorshubovych/markdownlint-cli:latest  "**/*.md"
```

Comment:
Is this a trustworthy image?

Reply:
yes it is, i know it does not look like it but in fact igorshubovych is the maintainer of markdownlint-cli